graphql.module

```diff
@@ -28,3 +28,32 @@ EOT;
 
   return $help;
 }
+
+/**
+ * Turn a list of machine names into a camel-cased string.
+ *
+ * @param string[]|string $components
+ *   Name components to be concatenated.
+ *
+ * @return string
+ *   A camel-cased concatenation of the input components.
+ */
+function graphql_camelcase($components) {
```

Comment:
Can you add another commit that changes all usages of graphql_core_camelcase and graphql_core_propcase to the new versions?